internal/pkg/v2/bootstrap/handlers/database/database.go

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package db
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg/bootstrap/interfaces"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/bootstrap/container"
+	db "github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure"
+	dbInterfaces "github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/interfaces"
+
+	bootstrapContainer "github.com/edgexfoundry/go-mod-bootstrap/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/startup"
+	bootstrapConfig "github.com/edgexfoundry/go-mod-bootstrap/config"
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+)
+
+// httpServer defines the contract used to determine whether or not the http httpServer is running.
+type httpServer interface {
+	IsRunning() bool
+}
+
+// Database contains references to dependencies required by the database bootstrap implementation.
+type Database struct {
+	httpServer httpServer
+	database   interfaces.Database
+	isCoreData bool
+}
+
+// NewDatabaseForCoreData is a factory method that returns an initialized Database receiver struct.
+func NewDatabaseForCoreData(httpServer httpServer, database interfaces.Database) Database {
```

Comment:
Again we have Core Data specific code in the common package. This should be in edgex-go/internal/core/data/v2/...

Reply:
Moved to `internal/core/data/v2/`.